### PR TITLE
Fixes link and image  markdown format and convert some HTMl to markdown.

### PR DESCRIPTION
**Pull request by chirag-wagento**
## This PR is a:
- [ ] New topic
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement
 

## Summary
 Fixes typo.
When this pull request is merged, it will...

### CODE REVIEW

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 1]
```diff
@@ -63,7 +65,7 @@ project.MyClass.prototype.method&nbsp;=&nbsp;function(foo)&nbsp;{
 };
 </pre>
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-classcomments">Class comments</h3>
+### Class comments
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-classcomments}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 2]
```diff
@@ -78,7 +80,7 @@ project.MyClass&nbsp;=&nbsp;function(arg1,&nbsp;arg2)&nbsp;{
 };
 </pre>
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-methodcomments">Method and function comments</h3>
+### Method and function comments
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-methodcomments}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 3]
```diff
@@ -94,7 +96,7 @@ function&nbsp;someMethod(obj)&nbsp;{
 }
 </pre>
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-propertycomments">Property comments</h3>
+### Property comments
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-propertycomments}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 4]
```diff
@@ -104,9 +106,9 @@ function&nbsp;someMethod(obj)&nbsp;{
 project.MyClass.prototype.someProperty&nbsp;=&nbsp;4;
 </pre>
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-tagreference">JSDoc tag reference</h3>
+### JSDoc tag reference
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-tagreference}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 5]
```diff
@@ -104,9 +106,9 @@ function&nbsp;someMethod(obj)&nbsp;{
...
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-const">@const</h4>
+#### @const
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-const}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 6]
```diff
@@ -123,7 +125,7 @@ Marks a variable read-only and suitable for inlining. Generates warnings if it i
 mynamespace.DEFAULT_TIMEZONE&nbsp;=&nbsp;'GMT';
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-extends">@extends</h4>
+#### @extends
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-extends}` to preserve the custom id

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 7]
```diff
@@ -139,7 +141,7 @@ project.MyClass.EmptyNodeList&nbsp;=&nbsp;function()&nbsp;{
 </pre>
 
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-interface">@interface</h4>
+#### @interface
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-interface}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 8]
```diff
@@ -161,7 +163,7 @@ function&nbsp;Polygon()&nbsp;{};
 Polygon.prototype.getSides&nbsp;=&nbsp;function()&nbsp;{};
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-implements">@implements</h4>
+#### @implements
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-implements}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 9]
```diff
@@ -183,7 +185,7 @@ Square.prototype.draw&nbsp;=&nbsp;function()&nbsp;{
 };
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-lends">@lends</h4>
+#### @lends
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-lends}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 10]
```diff
@@ -199,7 +201,7 @@ Please refer to <a href ="https://code.google.com/p/jsdoc-toolkit/wiki/TagLends"
 );
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-override">@override</h4>
+#### @override
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-override}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 11]
```diff
@@ -212,7 +214,7 @@ project.SubClass.prototype.toString()&nbsp;{
 };
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-param">@param</h4>
+#### @param
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-param}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 12]
```diff
@@ -230,7 +232,7 @@ Type names must be enclosed in curly braces. If the type is omitted, the compile
 };
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-return">@return</h4>
+#### @return
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-return}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 13]
```diff
@@ -245,7 +247,7 @@ namespace.Storage.prototype.getLastId&nbsp;=&nbsp;function()&nbsp;{
 };
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-this">@this</h4>
+#### @this
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-this}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 14]
```diff
@@ -262,7 +264,7 @@ The type of the object in whose context a particular method is called. Required
 );
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-type">@type</h4>
+#### @type
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-type}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 15]
```diff
@@ -274,7 +276,7 @@ Identifies the type of a variable, property, or expression.
 var&nbsp;hexId&nbsp;=&nbsp;hexId;
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-typedef">@typedef</h4>
+#### @typedef
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-typedef}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 16]
```diff
@@ -288,7 +290,7 @@ namespace.readNumber&nbsp;=&nbsp;function(x)&nbsp;{
 </pre>
 
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-jstypes">JavaScript types</h3>
+### JavaScript types
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-jstypes}` to preserve the custom id.

File: guides/v2.1/coding-standards/docblock-standard-javascript.md [hunk 17]
```diff
@@ -457,7 +459,7 @@ new&nbsp;project.MyClass()</pre></td>
   </tbody>
 </table>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-typelang">JavaScript type language</h4>
+#### JavaScript type language
```
Comment (jcalcaben) [inline guides/v2.1/coding-standards/docblock-standard-javascript.md]: Please append `{#frontend-dev-guide-javascript-js-coding-docblock-typelang}` to preserve the custom id.

File: guides/v2.1/comp-mgr/module-man/compman-checklist.md [hunk 18]
```diff
@@ -12,19 +12,19 @@ functional_areas:
   - Upgrade
 ---
 
-<h2 id="compman-overview">Overview of the Component Manager</h2>
+Overview of the Component Manager
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-checklist.md]: `h2` headings should be converted into the following format
```
## Overview of the Component Manager {#compman-overview}
```

File: guides/v2.1/comp-mgr/module-man/compman-main-pg.md [hunk 19]
```diff
@@ -12,11 +12,12 @@ functional_areas:
   - Upgrade
 ---
 
-<h2 id="compman-access">Component Management options</h2>
+Component Management options
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-main-pg.md]: `h2` headings should be converted into the following format
```
## Component Management options {#compman-access}
```

File: guides/v2.1/comp-mgr/module-man/compman-main-pg.md [hunk 20]
```diff
@@ -60,7 +61,8 @@ To choose whether to upgrade or uninstall components:
 	</tbody>
 	</table>
 
-<h2 id="compman-access-new">Install new purchases</h2>
+Install new purchases
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-main-pg.md]: `h2` headings should be converted into the following format
```
## Install new purchases {#compman-access-new}
```

File: guides/v2.1/comp-mgr/module-man/compman-main-pg.md [hunk 21]
```diff
@@ -86,44 +88,47 @@ To install a new purchase:
 
 	<img src="{{ site.baseurl }}/common/images/compman_new-purchases2.png" width="700px">
 3.	Click **Install**.
-4.	Continue with <a href="{{ page.baseurl }}/comp-mgr/module-man/compman-readiness.html">Readiness check</a>.
+4.	Continue with [Readiness check]({{ page.baseurl }}/comp-mgr/module-man/compman-readiness.html).
 
 
 
-<h2 id="compman-access-sync">Update a component</h2>
+Update a component
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-main-pg.md]: `h2` headings should be converted into the following format
```
## Update a component {#compman-access-sync}
```

File: guides/v2.1/comp-mgr/module-man/compman-main-pg.md [hunk 22]
```diff
@@ -86,44 +88,47 @@ To install a new purchase:
...
 The following figure shows an example of disabling a component.
 
-<img src="{{ site.baseurl }}/common/images/cman_actions-disable.png" alt="Click Disable from the Actions menu">
+![Click Disable from the Actions menu]({{ site.baseurl }}/common/images/cman_actions-disable.png)
 
-Continue with <a href="{{ page.baseurl }}/comp-mgr/module-man/compman-readiness.html">Readiness check</a>.
+Continue with [Readiness check]({{ page.baseurl }}/comp-mgr/module-man/compman-readiness.html).
 
-<h2 id="compman-uninst">Uninstall a component</h2>
+Uninstall a component
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-main-pg.md]: `h2` headings should be converted into the following format
```
## Uninstall a component {#compman-uninst}
```

File: guides/v2.1/comp-mgr/module-man/compman-main-pg.md [hunk 23]
```diff
@@ -86,44 +88,47 @@ To install a new purchase:
...
-<img src="{{ site.baseurl }}/common/images/cman_uninstall1.png" alt="Uninstall a component">
+![Uninstall a component]({{ site.baseurl }}/common/images/cman_uninstall1.png)
 
-Continue with <a href="{{ page.baseurl }}/comp-mgr/module-man/compman-readiness.html">Readiness check</a>.
+Continue with [Readiness check]({{ page.baseurl }}/comp-mgr/module-man/compman-readiness.html).
 
 
 
-<h2 id="compman-access-types">Supported actions for each component type</h2>
+Supported actions for each component type
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-main-pg.md]: `h2` headings should be converted into the following format
```
## Supported actions for each component type {#compman-access-types}
```

File: guides/v2.1/comp-mgr/module-man/compman-main-pg.md [hunk 24]
```diff
@@ -138,15 +143,13 @@ You can perform actions on components differently, depending on whether the comp
 
 The following sections provide details:
 
-*	<a href="#compman-access-types-meta">Actions available for metapackage components</a>
-*	<a href="#compman-access-types-non-meta">Actions available for non-metapackage components</a>
+*	[Actions available for metapackage components](#compman-access-types-meta)
+*	[Actions available for non-metapackage components](#compman-access-types-non-meta)
 
-<h3 id="compman-access-types-meta">Actions available for metapackage components</h3>
+### Actions available for metapackage components
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-main-pg.md]: Please append `{#compman-access-types-meta}` to preserve the custom id

File: guides/v2.1/comp-mgr/module-man/compman-start.md [hunk 25]
```diff
@@ -15,13 +15,14 @@ functional_areas:
   - Upgrade
 ---
 
-<h2 id="compman-access">Start the Component Manager from the Magento Admin</h2>
+Start the Component Manager from the Magento Admin
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-start.md]: `h2` headings should be converted into the following format
```
## Start the Component Manager from the Magento Admin {#compman-access}
```

File: guides/v2.1/comp-mgr/module-man/compman-main-pg.md [hunk 26]
```diff
@@ -138,15 +143,13 @@ You can perform actions on components differently, depending on whether the comp
...
 Magento requires any component that has more than one type to be a metapackage. For example, we require a component that contains a module and a {% glossarytooltip d2093e4a-2b71-48a3-99b7-b32af7158019 %}theme{% endglossarytooltip %} to be packaged as a metapackage.
 
-<div class="bs-callout bs-callout-info" id="info">
-      <p>We currently do not support uninstalling a metapackage.</p>
-</div>
+We currently do not support uninstalling a metapackage.
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-main-pg.md]: This conversion should look like the following to preserve the callout box:
```
{:.bs-callout .bs-callout-info}
We currently do not support uninstalling a metapackage.
```

File: guides/v2.1/comp-mgr/module-man/compman-main-pg.md [hunk 27]
```diff
@@ -195,11 +198,9 @@ The following table shows which actions are available for components that are pa
 		</tbody>
 	</table>
 
-<div class="bs-callout bs-callout-info" id="info">
-      <p><strong>Update</strong> displays as an action only if an update is available for that component.</p>
-</div>
+**Update** displays as an action only if an update is available for that component.
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-main-pg.md]: This conversion should look like the following to preserve the callout box:
```
{:.bs-callout .bs-callout-info}
**Update** displays as an action only if an update is available for that component.
```

File: guides/v2.1/comp-mgr/module-man/compman-main-pg.md [hunk 28]
```diff
@@ -195,11 +198,9 @@ The following table shows which actions are available for components that are pa
...
 
-<h3 id="compman-access-types-non-meta">Actions available for non-metapackage components</h3>
+### Actions available for non-metapackage components
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-main-pg.md]: Please append `{#compman-access-types-non-meta}` to preserve the custom id.

File: guides/v2.1/comp-mgr/module-man/compman-checklist.md [hunk 29]
```diff
@@ -12,19 +12,19 @@ functional_areas:
...
 This section discusses how to start the Component Manager, which updates or uninstalls Magento components:
 
 *	Modules (extend Magento capabilities)
 *	Themes (change the look and feel of your {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %} and Admin)
 *	Language packages (localize the storefront and Admin)
 
-<div class="bs-callout bs-callout-warning">
-    <p>If you installed the Magento application by <a href="{{ page.baseurl }}/install-gde/prereq/dev_install.html">cloning the GitHub repository</a>, you <em>cannot</em> use the Component Manager utility to update components. Instead, you must <a href="{{ page.baseurl }}/install-gde/install/cli/dev_options.html">update them manually</a>.</p>
-</div>
+If you installed the Magento application by [cloning the GitHub repository]({{ page.baseurl }}/install-gde/prereq/dev_install.html), you _cannot_ use the Component Manager utility to update components. Instead, you must [update them manually]({{ page.baseurl }}/install-gde/install/cli/dev_options.html).
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-checklist.md]: This conversion should look like the following to preserve the callout box:
```
{:.bs-callout .bs-callout-warning}
If you installed the Magento application by [cloning the GitHub repository]({{ page.baseurl }}/install-gde/prereq/dev_install.html), you _cannot_ use the Component Manager utility to update components.
Instead, you must [update them manually]({{ page.baseurl }}/install-gde/install/cli/dev_options.html).
```

File: guides/v2.1/comp-mgr/module-man/compman-checklist.md [hunk 30]
```diff
@@ -12,19 +12,19 @@ functional_areas:
...
 
-<h2 id="compman-prereq">Prerequisites</h2>
-Before continuing, complete all tasks discussed in <a href="{{ page.baseurl }}/comp-mgr/prereq/prereq_compman.html">Prerequisites</a>.
+Prerequisites
```
Comment (jcalcaben) [inline guides/v2.1/comp-mgr/module-man/compman-checklist.md]: `h2` headings should be converted into the following format
```
## Prerequisites {#compman-prereq}
```

**Comment by chirag-wagento** [comment]
Hello @jcalcaben  

I have make changes as per your advice.

Thanks

**Comment by jcalcaben** [comment]
Thanks for the quick turnaround on those changes @chirag-wagento !

**Pull request merged**